monitor/ui/pom.xml

```diff
@@ -211,7 +211,14 @@
 			<artifactId>DataCleaner-monitor-sugarcrm-datastore-wizard</artifactId>
 			<version>${project.version}</version>
 		</dependency>
-		
+
+        <!-- DataCleaner monitor documentation -->
+        <dependency>
+            <groupId>org.eobjects.datacleaner</groupId>
+            <artifactId>DataCleaner-monitor-documentation</artifactId>
```

Comment:
This dependency means that we are still not in a good situation. Combined with PR #679 you can see that this still introduces the banned dependencies.